lib/modules/dosomething/dosomething_zendesk/dosomething_zendesk.module

```diff
@@ -351,6 +351,18 @@ function dosomething_zendesk_verify_zendesk_user($email, $client = NULL) {
       // Create a zendesk user.
       return dosomething_zendesk_create_zendesk_user($email, $client);
     }
+    // Update user with profile fields.
+    $uid = user_load_by_mail($email)->uid;
+    $profile_fields = dosomething_zendesk_get_profile_fields($uid);
+    $id = $search->results[0]->id;
+    $updated_user = array(
+      'id' => $id,
+      'user_fields' => array(
+        'profile_url' => $profile_fields['profile_url'],
+        'profile_uid' => $profile_fields['profile_uid'],        
+      ),
+    );
+    $client->users()->update($updated_user);
```

Comment:
Can we check to see if the ZD User already has the fields set, vs setting every time? These values will never change, so its unnecessary overhead to post the update every time someone submits the form.